BUILD

```diff
@@ -177,11 +176,11 @@ objc_library(
     sdk_dylibs = ["libz"],
     sdk_frameworks = ["IOKit"],
     deps = [
-        "@FMDB//:fmdb",
-        "@MOLAuthenticatingURLSession//:MOLAuthenticatingURLSession",
-        "@MOLCodesignChecker//:MOLCodesignChecker",
-        "@MOLFCMClient//:MOLFCMClient",
-        "@MOLXPCConnection//:MOLXPCConnection",
+        "@FMDB",
```

Comment:
This fails - the rule inside the FMDB package is fmdb not FMDB.

Reply:
Did you see the change to the WORKSPACE? It works for me...and Travis.

Reply:
Oh the WORKSPACE change, nevermind.